packages/template-cli/templates/src/ssr/components/ApiPane/__snapshots__/index.test.tsx.snap

```diff
@@ -3,13 +3,13 @@
 exports[`renders ApiPane snapshot 1`] = `
 <DocumentFragment>
   <div
-    class="MuiGrid-root css-1hq8d8z MuiGrid-item"
+    class="MuiGrid-root css-t7xmz MuiGrid-item"
   >
-    <h1
-      class="MuiTypography-root MuiTypography-h1"
+    <h2
```

Comment:
This fails the aXe tests, you need at least one h1 on the page for accessibility. https://amido-dev.visualstudio.com/Amido-Stacks/_build/results?buildId=6304&view=logs&j=95fd2405-9ee1-5ffd-0e58-c04b67633907&t=07091dd7-b0ea-5595-026e-785d64cd745f